src/static/js/filter.js

```diff
@@ -0,0 +1,115 @@
+var $ = require('jquery');
+
+function isFilterSet(filterName) {
+  return document.cookie.search(filterName) >= 0;
+}
+
+module.exports = {
+  getFilterName: function(filterSelector, filters) {
+    for (var key in filters) {
```

Comment:
Don't use `for..in` with JavaScript arrays: https://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-a-bad-idea